src/main/java/com/michalsydoryk/app/gameengine/GameBuilder.java

```diff
@@ -0,0 +1,16 @@
+package com.michalsydoryk.app.gameengine;
+
+import com.michalsydoryk.app.board.Board;
+import com.michalsydoryk.app.board.Board2D;
+import com.michalsydoryk.app.boardchecker.Board2DChecker;
+
+public class GameBuilder {
+
+    public static Game build(GameConfiguration gameConfiguration){
+        Board board = gameConfiguration.board;
+        Game game = new Game(board, new Board2DChecker(board),
```

Comment:
One argument - one line -> would be easier to read

Reply:
done